yaksh/migrations/0006_auto_20170424_1015.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2017-04-24 10:15
+from __future__ import unicode_literals
+
+from django.db import migrations, models
```

Comment:
We need to finalise a naming convention ASAP and update all PRs accordingly, please provide suggestions here https://github.com/FOSSEE/online_test/issues/278

I will rename as per the final decision made in the above issue.